Description: 3.16.2 was released with a failing reftest. Let's not run it.
Author: Iain Lane <iain.lane@canonical.com>
Forwarded: not-needed

--- gtk+3.0-3.16.2.orig/testsuite/reftests/Makefile.am
+++ gtk+3.0-3.16.2/testsuite/reftests/Makefile.am
@@ -310,9 +310,6 @@ testdata = \
 	label-text-shadow-clipping.css \
 	label-text-shadow-clipping.ref.ui \
 	label-text-shadow-clipping.ui \
-	label-text-shadow-changes-modify-clip.css \
-	label-text-shadow-changes-modify-clip.ref.ui \
-	label-text-shadow-changes-modify-clip.ui \
 	label-width-chars-dont-shrink.ref.ui \
 	label-width-chars-dont-shrink.ui \
 	label-wrap-justify.ref.ui \
@@ -491,7 +488,6 @@ libreftest_la_CFLAGS = $(gtk_reftest_CFL
 libreftest_la_LIBADD = $(gtk_reftest_LDADD)
 libreftest_la_SOURCES =			\
 	expand-expander.c		\
-	label-text-shadow-changes-modify-clip.c	\
 	set-default-direction.c		\
 	statusbar-remove-all.c		\
 	textview-border-windows.c	\
